web-api/terraform/modules/api/alarms.tf

```diff
@@ -0,0 +1,25 @@
+
+resource "aws_cloudwatch_log_metric_filter" "pdf_parse_error_filter" {
+  name           = "PDFParseErrorFilter_${var.environment}_${var.current_color}"
+  log_group_name = "/aws/lambda/api_async_${var.environment}_${var.current_color}"
+  pattern        = "\"Failed to parse PDF\""
+
+  metric_transformation {
+    name      = "PDFParseErrors"
+    namespace = "EFCMS"
+    value     = "1"
+  }
+}
+
```

Comment:
Have you been able to confirm that this alarm is stable for a minimum of a week in test? 

A noisy alarm can be worse than no alarm.

Reply:
It's been in test since 6/27, so nearly a week, and seems to not be noisy. Totally agree that a noisy alarms can do more damage than having none 👍🏻